### ISSUE TITLE
A11y_SQL Azure Data Studio_NewTable_Cloumns_Labels Or Instructions:NVDA is not announcing the instructions provided by labels.

### ISSUE THREAD

**Issue by yoginigorle** [microsoft/azuredatastudio#19415]
#### GitHub Tags:
#A11yMAS;#A11yTCS;#Win10;#WCAG3.3.2;#DesktopApp;#A11ySev2;#NVDA;#SH_SQL Azure Data Studio_May2022;#SQL Azure Data Studio;

## Environmental Details:
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0

## OS Details :
Microsoft Windows 10

## Repro Steps:
1.Add connection to the server with appropriate credentials.
2.Tab till Servers in left navigation Bar.
3.Tab till tables and Press enter
4.Tab till New tables and press enter
5.Tab till column and press enter.
6.Turn on the NVDA.
7.Check the instructions provided by the labels under column properties are announcing by the NVDA or not.

## Note:
Same issue found for all the labels under properties of the Primary key,Foriegn keys,Check constraints,Indexes,General tab items.

## Actual results:
NVDA is not announcing the instructions provided by labels.

## Expected results:
NVDA should announce the instructions provided by the labels.

## User Impact:
Screen reader users not able to understand the purpose of the labels.

## Attachments:
https://user-images.githubusercontent.com/105736242/169036734-a54e85d1-e5fb-469c-b2de-aa9be8188aca.mp4

**Comment by Vishnuvarthancr7** [comment]
#Closed
<img width="960" alt="Screenshot (881)" src="https://user-images.githubusercontent.com/124578655/216965185-04809e40-bf1e-4722-bf31-f64ec52b8518.png">

**Comment by ManishaC1** [comment]
Github tags:#Closed;

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;
